.eslintrc.js

```diff
@@ -149,5 +149,18 @@ module.exports = {
         'jsdoc/require-jsdoc': 'off',
       },
     },
+    {
+      files: ['tests/*'],
```

Comment:
The tests folder is not linted, but eslint shows errors while editing these files, so I cleaned them up.